### DISCUSSION TITLE
RedisException Connection refused

### DISCUSSION THREAD

**Discussion by rajashri-samal90** [spatie/laravel-health#149] [General]
I am usong spatie-/laravel-health for checking Database and Redis Connectivity. If Redis server is down then its thowing RedisException Connection refused in vendor/laravel/framework/src/Illuminate/Redis/Connectors/PhpRedisConnector.php:153

**Comment by WayneBrummer** [comment]
I am using it n the same capacity with the same results. Since the RedisManager 
And honestly it makes the `RedisCheck::new(),` useless.  


Has anyone been able to figure this out?

**Comment by chris-lee-lb** [comment]
This is because this package need use cache driver to store notification histories for throttling check.
And if you also use Redis as your cache driver, of course you will face this problem.

The quick workaround maybe change your cache driver during the runtime, for example :
`CACHE_DRIVER=file php artisan health:check`

Or change your config `health.notifications.throttle_notifications_for_minutes` to 0 for always send notification without throttling check

Of course, this definitely will have side effect, use with caution.
